src/main/java/racing/Car.java

```diff
@@ -0,0 +1,29 @@
+package racing;
+
+public class Car implements Comparable<Car>{
+    private String name;
+    private int movingRange;
+
+    public Car(String name,int movingRange) {
+        if (name.length() > 5) throw new IllegalArgumentException("이름은 5자 이내로 .");
+        this.name = name;
+        this.movingRange = movingRange;
+    }
+    public Car(String name) {
+        this(name, 0);
+    }
+
+    public String getName() {
+        return name;
+    }
+    public int getMovingRange() {
+        return movingRange;
+    }
+
+    public void moved() { movingRange += 1; }
+
+    public int compareTo(Car o) {
```

Comment:
```suggestion
    @Override
    public int compareTo(Car o) {
```
코드컨벤션을 지켜주세요! 

[Google-code-style](https://google.github.io/styleguide/javaguide.html#s6.1-override-annotation)

Reply:
done